Description: Typo in git version macro
Author: Jörg Fringy-Fürst <debian@jff.email>
Forwarded: https://bugzilla.gnome.org/show_bug.cgi?id=795719
Last-Update: 2018-05-01
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: trunk/src/Resources.vala
===================================================================
--- trunk.orig/src/Resources.vala
+++ trunk/src/Resources.vala
@@ -18,7 +18,7 @@ namespace Resources {
     public const string APP_DIRECT_ROLE = _("Photo Viewer");
     public const string APP_VERSION = _VERSION;
 
-#if _GITVERSION
+#if _GIT_VERSION
     public const string? GIT_VERSION = _GIT_VERSION;
 #else
     public const string? GIT_VERSION = null;
